config/configuration.yml

```diff
@@ -21,15 +21,10 @@ default:
   email_delivery:
     delivery_method: :smtp
     smtp_settings:
-      enable_starttls_auto: <%= ENV['SMTP_ENABLE_STARTTLS_AUTO'] %>
+      enable_starttls_auto: <%= ENV['SMTP_ENABLE_STARTTLS_AUTO'] || true %>
       address: <%= ENV['SMTP_ADDRESS'] %>
-      port: <%= ENV['SMTP_PORT'] %>
+      port: <%= ENV['SMTP_PORT'] || 587 %>
       domain: <%= ENV['SMTP_DOMAIN'] %>
-      authentication: <%= ENV['SMTP_AUTHENTICATION'] %>
+      authentication: :plain
```

Comment:
@dkastl (CC: @smellman)
Well, after I read the following Redmine Email Configuration again, I prefer to keep this `SMTP_AUTHENTICATION` configurable.
https://www.redmine.org/projects/redmine/wiki/EmailConfiguration

Is there some reason to drop this configuration ?